Gemfile.jruby-3.1.lock.release

```diff
@@ -20,7 +20,7 @@ PATH
       manticore (~> 0.6)
       minitar (~> 0.8)
       pry (~> 0.12)
-      puma (~> 6.3, >= 6.3.1)
+      puma (~> 6.3, >= 6.4.2)
```

Comment:
interesting... this update should be for minor versions.
Any ideas @andsel @jsvd about safety to apply this change?

Reply:
After a cursory glance at the changes it seems ok to upgrade. we will also want to upgrade to that version due to an important fix in 6.4.2